src/main/java/org/cbioportal/web/util/InvolvedCancerStudyExtractorInterceptor.java

```diff
@@ -76,17 +67,23 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.web.servlet.HandlerInterceptor;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
 public class InvolvedCancerStudyExtractorInterceptor implements HandlerInterceptor {
 
     @Autowired
     private UniqueKeyExtractor uniqueKeyExtractor;
 
     private ObjectMapper objectMapper = new ObjectMapper();
 
-    @Qualifier("staticRefCacheMapUtil")
```

Comment:
Why is this one no longer necessary?

Reply:
That was there to force to a specific class when multiple IMPLs are available for a specific Interface. After adding the Conditionals for the CacheMapUtils there is now only one CacheMapUtils Implementation on the class path.